Makefile

```diff
@@ -99,93 +77,106 @@ else
 
-testwhisper: | build deps
-	echo -e $(BUILD_MSG) "build/$@" && \
-		$(ENV_SCRIPT) nim testwhisper $(NIM_PARAMS) waku.nims
+### RLN
 
+LIBRLN_BUILDDIR := vendor/zerokit/target/release
 
-## Waku v1 targets
+ifeq ($(OS),Windows_NT)
+LIBRLN_FILE := rln.dll
+else ifeq ($(strip $(shell uname)),Darwin)
+LIBRLN_FILE := librln.dylib
+else
+LIBRLN_FILE := librln.so
+endif
 
-test1: | build deps
-	echo -e $(BUILD_MSG) "build/$@" && \
-		$(ENV_SCRIPT) nim test1 $(NIM_PARAMS) waku.nims
+$(LIBRLN_BUILDDIR)/$(LIBRLN_FILE):
+	echo -e $(BUILD_MSG) "$@" && \
+		cargo build --manifest-path vendor/zerokit/rln/Cargo.toml --release
 
-wakunode1: | build deps
-	echo -e $(BUILD_MSG) "build/$@" && \
-		$(ENV_SCRIPT) nim wakunode1 $(NIM_PARAMS) waku.nims
+ifneq ($(RLN), true)
+librln: ; # noop
+else
+EXPERIMENTAL_PARAMS += -d:rln -d:rln_libpath=$(LIBRLN_BUILDDIR)/$(LIBRLN_FILE)
+librln: $(LIBRLN_BUILDDIR)/$(LIBRLN_FILE)
+endif
 
-example1: | build
-	echo -e $(BUILD_MSG) "build/$@" && \
-		$(ENV_SCRIPT) nim example1 $(NIM_PARAMS) waku.nims
+clean-librln:
+	cargo clean --manifest-path vendor/zerokit/rln/Cargo.toml
 
-sim1: | build deps wakunode1
-	echo -e $(BUILD_MSG) "build/$@" && \
-		$(ENV_SCRIPT) nim sim1 $(NIM_PARAMS) waku.nims
+# Extend clean target
+clean: | clean-librln
 
 
-## Waku v2 targets
+#############
+## Waku v2 ##
+#############
+.PHONY: test2 wakunode2 example2 sim2 scripts2 wakubridge chat2 chat2bridge
 
-test2: | build deps deps2
+test2: | build deps librln
 	echo -e $(BUILD_MSG) "build/$@" && \
-		$(ENV_SCRIPT) nim test2 $(NIM_PARAMS) waku.nims
+		$(ENV_SCRIPT) nim test2 $(NIM_PARAMS) $(EXPERIMENTAL_PARAMS) waku.nims
 
-wakunode2: | build deps deps2
+wakunode2: | build deps librln
 	echo -e $(BUILD_MSG) "build/$@" && \
-		$(ENV_SCRIPT) nim wakunode2 $(NIM_PARAMS) waku.nims
+		$(ENV_SCRIPT) nim wakunode2 $(NIM_PARAMS) $(EXPERIMENTAL_PARAMS) waku.nims
 
 example2: | build deps
 	echo -e $(BUILD_MSG) "build/$@" && \
 		$(ENV_SCRIPT) nim example2 $(NIM_PARAMS) waku.nims
 
-sim2: | build deps deps2 wakunode2
+# TODO: Remove unused target
+sim2: | build deps wakunode2
 	echo -e $(BUILD_MSG) "build/$@" && \
 		$(ENV_SCRIPT) nim sim2 $(NIM_PARAMS) waku.nims
 
+# TODO: Remove unused target
 scripts2: | build deps wakunode2
 	echo -e $(BUILD_MSG) "build/$@" && \
 		$(ENV_SCRIPT) nim scripts2 $(NIM_PARAMS) waku.nims
 
-wakubridge: | build deps deps2
+wakubridge: | build deps
 	echo -e $(BUILD_MSG) "build/$@" && \
 		$(ENV_SCRIPT) nim bridge $(NIM_PARAMS) waku.nims
 
-chat2: | build deps deps2
+chat2: | build deps librln
 	echo -e $(BUILD_MSG) "build/$@" && \
-		$(ENV_SCRIPT) nim chat2 $(NIM_PARAMS) waku.nims
+		$(ENV_SCRIPT) nim chat2 $(NIM_PARAMS) $(EXPERIMENTAL_PARAMS) waku.nims
 
-chat2bridge: | build deps deps2
+chat2bridge: | build deps
 	echo -e $(BUILD_MSG) "build/$@" && \
 		$(ENV_SCRIPT) nim chat2bridge $(NIM_PARAMS) waku.nims
 
-## Waku v2 tooling targets
+
+###################
+## Waku v2 tools ##
+###################
+.PHONY: wakucanary wakumonitor
```

Comment:
can we use `networkmonitor` instead of `wakumonitor` to be consistent with the target. https://github.com/waku-org/nwaku/pull/1437/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R185

Reply:
My bad, changing it 👍🏼